pkg/builder/build.go

```diff
@@ -206,7 +207,7 @@ func (b generalBuilder) build(ctx context.Context, pw progresswriter.Writer) err
 
 	for _, entry := range b.entries {
 		// Set up docker config auth.
-		attachable := []session.Attachable{authprovider.NewDockerAuthProvider(os.Stderr)}
+		attachable := []session.Attachable{authprovider.NewDockerAuthProvider(&configfile.ConfigFile{})}
```

Comment:
What does this change mean?

Reply:
the `authprovider` is upgraded with the `buildkit` client library, the NewDockerAuthProvider param is updated from upstream so that I have to change the args

Reply:
Gotcha.